cli/kv.go

```diff
@@ -300,7 +300,7 @@ func initScanArgs(args []string) (startKey, endKey proto.Key) {
 	if len(args) >= 2 {
 		endKey = proto.Key(unquoteArg(args[1], false))
 	} else {
-		endKey = proto.KeyMax
+		endKey = keys.TableDataPrefix
```

Comment:
The keys and values for table data are encoded and look like gibberish. Seems better to not show them by default and require the user to explicitly ask for them. That was the thought process behind this. I'm open to hearing arguments for leaving this unchanged.